test/module/shared_model/validators/protobuf/proto_query_validator_test.cpp

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "validators/protobuf/proto_query_validator.hpp"
```

Comment:
```C++
#include "validators/protobuf/proto_query_validator.hpp"

#include "module/shared_model/validators/validators_fixture.hpp"
#include "queries.pb.h"
```
since `proto_query_validator` is the header of the class being tested here